test/ssl-tests/20-cert-select.conf.in

```diff
@@ -859,6 +858,7 @@ my @tests_tls_1_3 = (
             "Certificate" => test_pem("server-ecdsa-brainpoolP256r1-cert.pem"),
             "PrivateKey" => test_pem("server-ecdsa-brainpoolP256r1-key.pem"),
             "Groups" => "brainpoolP256r1",
+            "MaxProtocol" => "TLSv1.2"
```

Comment:
It looks like we should introduce some new tests for brainpool in TLSv1.3

Reply:
I can add a second test here, but it does not seem to do the full handshake,
for tls1.2 yes, but tls1.3 does not have the server certificate ready when the
test expects to see it, the certificate is only there after the new session ticket arrives.

Reply:
I don't understand what is meant by "tls1.3 does not have the server certificate ready when the test expects to see it, the certificate is only there after the new session ticket arrives" -- in TLS 1.3, the NewSessionTicket message is not sent until after the server Finished, which is after the Certificate is sent (though can be in the same flight).

Reply:
Just a vague idea.  Since it is impossible for me to debug into this test case,
I tried to print the server certificate in s_client and the certificate was NULL
at the point where tls1.2 had it available, but very short after that there
is a new session callback and the certificate is available there.
It is funny that s_client does not seem to show the server certificate, since
I think I remember that was once the case.

Reply:
@kaduk I think everything woks as expected, I was just confused since this did not work:
```
         test   => {
+            "ExpectedClientCertType" => "brainpoolP256r1",
+            "ExpectedClientSignHash" => "SHA256",
+            "ExpectedClientSignType" => "EC",
             "ExpectedResult" => "Success"
```

I thought that would test the cert received by the client.
but instead it tests the client cert received by the server.
but since there is no client brainpool cert available that did not work.
No idea what I did wrong with the s_client test I cannot reproduce
what I saw there.

Reply:
Ah, thanks for clarifying.
I trust you found the `ExpectedserverCertType` and related directives that would apply for the server's certificate.